lib/middleware/fetch-woeid.js

```diff
@@ -0,0 +1,17 @@
+const { getWoeid } = require('../services/woeid');
+
+module.exports = (req, res, next) => {
+  const location = req.body.location;
+  try {
```

Comment:
When using promises use .catch instead of try/catch

```js
getWoeid(city)
      .then(woeid => {
        req.woeid = woeid;
        next();
      })
      .catch(err => {
         err.status = 400;
         next(err)
      });
```